fe/fe-core/src/test/java/com/starrocks/http/BadHttpRequestTest.java

```diff
@@ -0,0 +1,103 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.